### ISSUE TITLE
Automate the closure of icon requests from 'forbidden' brands

### ISSUE THREAD

**Issue by adamrusted** [simple-icons/simple-icons#9999]
As we've seen in #9969 - the number of brands we will not consider for this project is growing quite significantly.
The workflow introduced in #8695 specifically closes issues that feature 'Java' - regardless of casing. I feel like we can potentially develop an action which can take an array of strings, perhaps a file that also populates our README, and filters all new issues against them, marking them as 'assessing' (see [Discord discussion](https://discord.com/channels/1142044630909726760/1179519264630259795/1180122580728938537) on new tags) but not closing them.

**Comment by service-paradis** [comment]
I agree we could have something automated for those brands. Honestly, I still thinks we should automatically close Java. There is too many of those issues and we know we have to close them. Maybe we could have 2 categories, Brands we're sure we want to close and other we want to assess just in case?

**Comment by adamrusted** [comment]
There is the ongoing discussion internally around streamlining wherever we can. While I agree it'd be _easiest_ to continue automatically closing Java requests, I don't want us duplicating effort in multiple locations to achieve (essentially) the same thing.

One of us reviewing the `assessing` tag - even once a week - isn't too big a chore, which is why I proposed going that route.

**Comment by adamrusted** [comment]
How's about this, @service-paradis. We store (somewhere in this repo) a list of forbidden brands. That forbidden brands list feeds into an action that splits each string apart, generates a case-insensitive regex, and compares the incoming issue/PR against that.

**Comment by AnaBMamprin** [comment]
I'm sorry, but we really don't have the Java icon? I've searched and couldn't find... I need for my README.

**Comment by uncenter** [comment]
> I'm sorry, but we really don't have the Java icon? I've searched and couldn't find... I need for my README.

See #7374. We are not allowed to include the Java icon.

**Comment by AnaBMamprin** [comment]
> > I'm sorry, but we really don't have the Java icon? I've searched and couldn't find... I need for my README.
> 
> See #7374. We are not allowed to include the Java icon.

Oh, okay, I didn't find it before, thank you.

**Comment by PeterShaggyNoble** [comment]
Closing as this was resolved by #13022
